### DISCUSSION TITLE
[Issue]  Explain this ?

### DISCUSSION THREAD

**Discussion by PrestigeDevop** [Voltstro-Studios/UnityWebBrowser#312] [Q&A] [answered]
### What platform are you experiencing this issue on?

Windows

### What architecture is your platform?

64-Bit

### What version of UWB are you using?

Latest VoltstroUPM Release

### What Unity version are you running?

2021

### Describe what the issue you are experiencing is.

```

[Web Browser]: [CEF Engine Console]: An error occurred while trying to load 'https://accounts.google.com/gsi/button?theme=outline&text=signin_with&size=medium&type=standard&shape=rectangular&width=-48&logo_alignment=left&click_listener=function()%7BtrackButtonClick(r)%7D&client_id=657641920759-c7l1281jejtqqb0rr9jc03qgp9se6gms.apps.googleusercontent.com&iframe_id=gsi_195981_499872&as=AxgVbiSLwTnGk9O1OZGDPw'! Error details: ERR_ABORTED (Code: ABORTED)

UnityEngine.Logger:LogError (string,object)
VoltstroStudios.UnityWebBrowser.Logging.DefaultUnityWebBrowserLogger:Error (object) (at Library/PackageCache/dev.voltstro.unitywebbrowser@2.1.1/Runtime/Logging/DefaultUnityWebBrowserLogger.cs:36)
VoltstroStudios.UnityWebBrowser.Logging.ProcessLogHandler:HandleOutputProcessLog (object,System.Diagnostics.DataReceivedEventArgs) (at Library/PackageCache/dev.voltstro.unitywebbrowser@2.1.1/Runtime/Logging/ProcessLogHandler.cs:54)
System.Threading._ThreadPoolWaitCallback:PerformWaitCallback ()
```

I got this error for no reason , I haven't change the url ,so why there's calls to this endpoints ?  did you install addtional browser plugins ? 

why there's a call for google endpoint 

### Provide reproducible steps for this issue.

disable ui components 

### Any additional info you like to provide?

please explain it this sounds sus

**Comment by PrestigeDevop** [comment]
it might be irrelevant but does this has anything to do with Google's leaked ranking system ? 

for reference https://www.youtube.com/watch?v=XNQhDl4a9Ko&ab_channel=Fireship

**Comment by Voltstro** [comment, marked as answer]
First up, this is a question, not a bug report. Please only use Issues for bug reports.

> I got this error for no reason

Error is related to a request to Google's login endpoint being aborted for some reason. I do not know why. Could be for any number of reasons.

> did you install addtional browser plugins

This project just uses the standard build of CEF provided by the CEF build servers.

> I got this error for no reason , I haven't change the url ,so why there's calls to this endpoints ?

I don't quite understand what you mean by "I haven't change the url". And I do not know why there is a call to Google's login endpoint. Only assumption I could say is whatever site you are on is making the call. UWB runtime it self never makes any web calls.
